src/connectionconfig/connectionDialogWebviewController.ts

```diff
@@ -634,8 +636,31 @@ export class ConnectionDialogWebviewController extends FormWebviewController<
 
         try {
             try {
+                if (
+                    this.state.selectedInputMode ===
```

Comment:
I wonder if we should adjust the code formatting rules?  This 4 line if statement seems less readable than the normal ` line way we'd normally write this.

Reply:
100% agree that Prettier is wrapping too narrowly.  I'll send a PR to update the width.